tests/_serialization/test_taurus_interop.py

```diff
@@ -19,5 +19,5 @@ def test_flatten():
     )
     spec = ProcessSpec(name="spec", template=template)
 
-    flat = flatten(spec)
+    flat = flatten(spec, scope='testing')
```

Comment:
Calls to flatten now require an explicit choice of scope for dissecting the tree.